### PR TITLE
Add Mods Before Friday

### PR DESCRIPTION
**Pull request by WombleWoo7547**
Adds documentation for the mods before friday free online tool to mod Beat Saber on the Meta/Oculus Quest VR headsets.

### CODE REVIEW

File: wiki/quest-modding-modsbf.md
Comment (bloodcloak) [inline wiki/quest-modding-modsbf.md:1]: Content length doesn't seem to require this whole new page. Merge into a section in the main quest modding page `Ways to Mod Beat Saber` Section

File: wiki/quest-modding-modsbf.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+### Step 2: Go to MBF
+Go to [MBF](https://mbf.bsquest.xyz/) and click ***Connect to Quest***.
+Select your Meta Quest VR headset from the prompt.
+
+### Step 3: Do the thing.
+Follow the instructions on screen.
+Can you do that?
+ 👍 If yes good job you have a brain cell
+ 👎 If no then you should get checked out
```
Comment (bloodcloak) [inline wiki/quest-modding-modsbf.md:21]: Should remove as this tone does not match the wiki.

File: wiki/quest-modding-modsbf.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Quest modding using MBF
+:::danger Dev mode
+Developer mode needs to be enabled and the computer needs to be trusted.
+:::
+
+## What?
```
Comment (bloodcloak) [inline wiki/quest-modding-modsbf.md:6]: Refer to how the Quest Patcher section is written for structuring instructions to getting started with this method

File: wiki/quest-modding.md [hunk 3]
```diff
@@ -86,6 +86,10 @@ follow the steps below:
 
 ### SideQuest
 
+:::tip Mods Before Friday
+No need for SideQuest if using Mods Before Friday
+:::
+
 This is only required if you mod with a PC (Windows / Mac / Linux). Go to [SideQuest's Download Page](https://sidequestvr.com/setup-howto)
```
Comment (bloodcloak) [inline wiki/quest-modding.md:93]: Should revise this sentence in addition to not conflict as a PC is still required to use MBF

File: wiki/quest-modding.md [hunk 4]
```diff
@@ -138,6 +142,15 @@ latest moddable version.
 
 Visit [Modding With QAVS](./quest-modding-qavs.md) to get started.
 
+## Mods Before Friday
+
+Mods Before Friday is a WebUSB-based tool that lets you mod Beat Saber from a browser
+:::warning Supported Browsers
+Some browsers aren't supported and some say they are but then corrupt your Beat Saber.
```
Comment (bloodcloak) [inline wiki/quest-modding.md:149]: Source for this?

**Review by bloodcloak — CHANGES_REQUESTED** [review]

File: wiki/quest-modding.md (hunk 4 above)
Comment (Lauriethefish) [reply wiki/quest-modding.md:149]: It is seems unlikely that using an unsupported browser will corrupt your installation. MBF uses an agent (small executable) to perform any modifications on your quest and if there are issues with WebADB, that agent will probably get corrupted before it even runs and then fail to launch, so Beat Saber should never get damaged.

(I have never seen a report of this happening so it doesn't deserve to be included)

File: wiki/quest-modding.md (hunk 3 above)
Comment (Lauriethefish) [reply wiki/quest-modding.md:93]: A PC is not required to use MBF - it will run fine on an Android phone with an up-to-date browser.

**Comment by Lauriethefish** [comment]
Hey, MBF author here.  I'm sorry for not making this clear (by making a draft PR) but I already did a bunch of work on updating the wiki to include instructions to use MBF. My fork is [here](https://github.com/Lauriethefish/bsmg-wiki) and can be previewed [here](https://lauriethefish.github.io/bsmg-wiki/quest-modding).

My fork also updates the whole of the quest modding section to reflect the current state of things (i.e., no need to use questpatcher, QAVS and BMBF **do not work** etc...) so it might be a better bet in the long run. I will do a quick check over it then open a pull request later today if you don't mind.

Alternatively, if you have some ideas that you think my branch is lacking, LMK and I can incorporate.

**Pull request closed without merging**